src/Orangebeard.Client/OrangebeardV2Client.cs

```diff
@@ -0,0 +1,257 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Orangebeard.Client.Entities;
+using Orangebeard.Client.OrangebeardProperties;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orangebeard.Client
+{
+    public class OrangebeardV2Client : AbstractClient
+    {
+        //private readonly ILogger<OrangebeardV2Client> LOGGER;
+
+        private bool connectionWithOrangebeardIsValid = false;
+        private readonly OrangebeardConfiguration config;
+
+        private OrangebeardV2Client()
+        {
+            //var loggerFactory = (ILoggerFactory)new LoggerFactory();
+            //LOGGER = loggerFactory.CreateLogger<OrangebeardV2Client>();
+        }
+
+        private void Init(string endpoint, Guid uuid)
+        {
+            InitializeHttpClient(new Uri(endpoint), uuid.ToString(), null);
+
+            ServicePointManager.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true; //TODO?- Not sure if this is still necessary.
+            ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12;
+        }
+
+        public OrangebeardV2Client(string endpoint, Guid uuid, string projectName, string testSetName, bool connectionWithOrangebeardIsValid) : this()
```

Comment:
where does the connectionWithOrangebeardIsValid originate? should it not be a method call instead of a client-provided value?